chain-api/src/utils/signatures.ts

```diff
@@ -247,6 +276,13 @@ function signSecp256k1(dataHash: Buffer, privateKey: Buffer, useDer?: "DER"): st
 
   const signature = ecSecp256k1.sign(dataHash, privateKey);
 
+  // Low-S normalization
+  let newSig;
+  if (signature.s.cmp(ecSecp256k1.curve.n.shrn(1)) > 0) {
+    newSig = flipSignatureParity(signature);
+    newSig.toDER = signature.toDER;
+  }
+
```

Comment:
Was the implementation complete here? It seems nothing happens with `newSig`

```suggestion
  let signature = ecSecp256k1.sign(dataHash, privateKey);

  if (signature.s.cmp(ecSecp256k1.curve.n.shrn(1)) > 0) {
    signature = flipSignatureParity(signature);
  }

```

Reply:
The reason this is done this way is bc the newSig can be an Secp256k1Signature instead of EC.Signature, which doesn't have the toDER function on it. Maybe there's a better way of dealing with this type discrepancy